### DISCUSSION TITLE
elyra kubeflow API endpoint

### DISCUSSION THREAD

**Discussion by RosetteAli** [elyra-ai/elyra#1570] [Pipeline Editor/ Kubeflow Pipelines + Apache Airflow]
Hello , I need to ask about the pipeline API endpoint when defining the run time 
I have errors when running elyra on kubeflow in ibm cloud , where can i get the correct API endpoint needed for the configuration ?

**Comment by ptitzler** [comment]
Hi @RosetteAli, a couple of questions:

- Which version of Elyra did you install?
- What error are you getting?
- Did you deploy Kubeflow on the [IBM Kubernetes Service or on OpenShift on IBM Cloud](https://www.kubeflow.org/docs/ibm/deploy/)? 
- Are you able to access the Kubeflow or Kubeflow Pipelines web console outside of Elyra?

For IKS, did you complete these steps? https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow-on-iks/#expose-the-kubeflow-endpoint-as-a-loadbalancer

For OpenShift, take a look at the last section in https://www.kubeflow.org/docs/ibm/deploy/install-kubeflow-on-ibm-openshift/#single-user

Irrespective of whether you've deployed Kubeflow on IKS or OpenShift, make sure you have selected `Tekton` as [engine type in the runtime configuration](https://elyra.readthedocs.io/en/latest/user_guide/runtime-conf.html#kubeflow-pipelines-engine-engine). If you choose the wrong type, the pipeline will fail to execute.
